instaclone/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import './PostContianer.css';
+import PostHeader from './PostHeader/PostHeader';
+import CommentSection from './CommentSection/CommentSection';
+
+class PostContainer extends Component {
+	render() {
+		return this.props.appData.map(post => {
+			return (
+				<div key={Math.random()}>
```

Comment:
Check out the [React docs section on lists-and-keys](https://reactjs.org/docs/lists-and-keys.html). They have some good recommendations on how to pick a key to use for lists.